app/deed/view/routes.py

```diff
@@ -0,0 +1,61 @@
+import requests
+from app.deed.view import views
+from app.domain.borrower import Borrower
+from app.domain.lender import Lender
+from app.domain.land_property import LandProperty
+from flask import redirect
+
+
+def register_routes(blueprint):
+    deed_api_base_host = 'http://localhost:5050'
```

Comment:
This can't be guaranteed in every environment. Can you make this configurable through ENV_VARS? We have an `app/config.py` so you could pull them from the ENV there and then import and use them here.